Governance document (#2)
USRSE/documents: governance.md

# The United States Research Software Engineer (US-RSE) Association

## Membership and Governance

> Date: June XX, 2020

Author: ```suggestion
Date: June 29, 2020
```

the seat shall remain open until the end of the seat's term.

## Committees, Working Groups, and Community Involvement

Organizational functions of the US-RSE Association will be managed by standing committees that may
consist of one or more members of the Steering Committee and interested members of and participants in
the Association. These organizational activities include communications (website, social media,
etc.), conference planning, and others to be determined by the Steering Committee.

Other activities, events, and efforts that have a fixed end date will be managed by working groups.
These working groups can and should be initiated by members at large through discussion with the
Steering Committee to ensure alignment with US-RSE priorities.  Working groups should have at least
three (3) US-RSE members as participants.

## Changes to Governance

Minor revisions to this document can be made with the approval of at least two-thirds (⅔) of the current
steering committee members. Major changes to the governance structure or this document will require
a vote of the membership directly.

## Ratification

> This document was ratified by the following members of the First US-RSE Steering Committee on June
> XX, 2020.

Author: ```suggestion
This document was ratified by the following members of the First US-RSE Steering Committee on June
29, 2020.
```

  US-RSE Association umbrella technologies including but not limited to:
    * All repositories in the US-RSE GitHub organization
    * All mailboxes and mailing lists associated with the us-rse.org domain name
    * The us-rse.org domain and website
    * The US-RSE slack channel
    * Any other official accounts intended to represent the overall organization, now or in the
      future (e.g., Twitter, Facebook, LinkedIn, etc.)

The Steering Committee strives for consensus in its decisions. In the case of a dispute or argument
within the community, a majority vote of all of the Steering Committee members will result in a
final decision. In the event of a tie, the Steering Committee chair will cast the deciding vote.  At
all Committee meetings, a quorum is at least one half of the members.

### Steering Committee Elections and Terms

After the conclusion of the founding Steering Committee's tenure, all future Steering Committee
members will be elected by the general US-RSE membership.  Steering committee members are elected to
a 2-year term. To provide continuity, steering committee members will have staggered terms, with
four or five members elected each year.

> Nominations for open positions will be solicited from all members at least 6 weeks before the

Reviewer: I think it is important that we make an active effort to ensure "all" members are aware of the nomination process and how to participate. I had suggested putting in some more process details here - such as "an email will be sent out to all members informing them of the nomination procedures." I am fine keeping the wording as-is, but I want to emphasize the importance of making an effort to actively solicit the nominations from "all" members! In addition, the same active efforts should be made to boost participation in voting!

Reviewer 2: I agree with you, @jordanperr, we'll want to make lots of noise. I feel like we aren't a very "loud" organization, but leading up to the nomination/election we should be more persistent with emails, slack, etc.

Reviewer 3: I agree we should make plenty of effort to notify everyone, but I don't think this document is the right place for specifics of how we do that.  I suggest we leave the wording as it is, but use this PR discussion as a note to make sure we have a good announcement system.

Reviewer 4: I also don't think we need to say how in this document

Author: I agree to leave as is, esp. with the added clarification by Jordan in the announcement letter.

may choose to appoint someone to fill the position for the remainder of the term. If the Steering
Committee chooses to fill the position, the replacement will be chosen by a majority vote of the
remaining members of the Steering Committee.  In the event that a majority vote cannot be achieved,
the seat shall remain open until the end of the seat's term.

## Committees, Working Groups, and Community Involvement

Organizational functions of the US-RSE Association will be managed by standing committees that may
consist of one or more members of the Steering Committee and interested members of and participants in
the Association. These organizational activities include communications (website, social media,
etc.), conference planning, and others to be determined by the Steering Committee.

Other activities, events, and efforts that have a fixed end date will be managed by working groups.
These working groups can and should be initiated by members at large through discussion with the
Steering Committee to ensure alignment with US-RSE priorities.  Working groups should have at least
three (3) US-RSE members as participants.

## Changes to Governance

> Minor revisions to this document can be made with the approval of at least two thirds (⅔) of the current

Author: ```suggestion
Minor revisions to this document can be made with the approval of at least two-thirds (⅔) of the current
```
per @cmaimone

steering committee members. Major changes to the governance structure or this document will require
a vote of the membership directly.

## Ratification

This document was ratified by the following members of the First US-RSE Steering Committee on June
XX, 2020:

* Jeffrey C. Carver
* Ian Cosden (chair)
* Charles Ferenbaugh
* Sandra Gesing (vice-chair)
* Chris Hill
* Daniel S. Katz
* Christina Maimone (election chair)
* Lance Parsons

> * Jordan Perr-Sauer (secretary)

Reviewer: In addition to the author list, I think having such a "signatures" section may become cumbersome over time. What happens if a change is made down the road after one or more of us are no longer participating?

As this isn't a declaration of independence, I'm not sure what the value is of having our "signatures" here. Thoughts?

Reviewer 4: it's fine with me to remove it

Reviewer 2: I liked the signature section before this moved to github. The approvers will be clear so I think it's ok to remove it, but it's possible there is a good reason to have it in there. We should get agreement from everyone (including those who have already approved).

Author: I'm fine with either keeping it or removing it. It's probably simpler to remove.

Author: If we do remove it, we will need to add the positions that people have taken, (chair, vice-chair, election chair, and secretary) via the transition letter, and imho, a page on the website (perhaps with short bios).

Reviewer: There is already a page on the website. I suggest we simply keep the page up to date with the results of each election. It would also be nice if we keep a list of "former members of the steering committee" on that same web page - or easily accessible from that page.

https://us-rse.org/steering-committee/

Reviewer 4: agreed

Reviewer 2: Sounds good. Agree that we should put the roles on the webpage. I added chair months ago before we had the others.

# The United States Research Software Engineer (US-RSE) Association

## Membership and Governance

> Date: June XX, 2020

Reviewer 2: How about we just pick today? 
```suggestion
Date: June 12, 2020
```

Author: Perhaps the date we get everyone to approve?

Reviewer: I suggest we use the date this is merged into the repo. Whoever does the merge should just make one final commit to update this date.

Reviewer: This date would be updated every time there is a merge.

Reviewer 2: OK. I like the suggestion that whoever merges updates the date.

> Authors: The founding members of the US-RSE Association Steering Committee

Reviewer 4: should we list our names here?

Author: That seems fine by me.

Reviewer: Having a list of authors could become cumbersome over time, especially since we've allowed revisions to this document. Is there a need to have an author list?

Reviewer 4: If you want to just remove the line "Authors: The founding members of the US-RSE Association Steering Committee", that would be fine with me

Reviewer 2: Now that you bring it up, I agree that it could get cumbersome. I'm fine with removing the "Authors: .." line too.

### Organizational Mission

The United States Research Software Engineer (US-RSE) Association is an open organization that
strives to:

* provide a coherent professional community in the United States for those who identify as a
    Research Software Engineer (RSE),
* advocate for RSE professionals, and
* provide resources to support and advance RSEs and the RSE career path within the United States
    research ecosystem.

### Code of Conduct

The US-RSE Association requires all members to adhere to the
[US-RSE Code of Conduct](https://us-rse.org/code-of-conduct/) in all US-RSE activities, forums,
and communications.

### Membership

> Membership is open to anyone who supports the mission of the Association and who agrees to abide by

Reviewer 4: ```suggestion
Membership is open to anyone who supports the mission of the US-RSE Association and who agrees to abide by
```

  Association, based on the inputs of the US-RSE Association membership.
* The Steering Committee is ultimately responsible for day-to-day activities, operations, and
  development, but many of these actions and activities will be initiated, organized, and managed by
  community committees and members at large.
* Members of the Steering Committee have the ability to grant or remove administrative rights for
  US-RSE Association umbrella technologies including but not limited to:
    * All repositories in the US-RSE GitHub organization
    * All mailboxes and mailing lists associated with the us-rse.org domain name
    * The us-rse.org domain and website
    * The US-RSE slack channel
    * Any other official accounts intended to represent the overall organization, now or in the
      future (e.g., Twitter, Facebook, LinkedIn, etc.)

The Steering Committee strives for consensus in its decisions. In the case of a dispute or argument
within the community, a majority vote of all of the Steering Committee members will result in a
final decision. In the event of a tie, the Steering Committee chair will cast the deciding vote.  At

> all Committee meetings, quorum is at least one half of the members.

Reviewer 4: ```suggestion
all Committee meetings, a quorum is at least one half of the members.
```

election.  The list of nominated candidates will be announced at least 3 weeks before the election.
All US-RSE members in good standing on the date that the nominations are opened are eligible to be
nominated as candidates.

### Steering Committee Meetings and Communication

Steering Committee meetings will take place at a minimum of once per quarter, and typically will be
held monthly. All Steering Committee meetings will have notes recorded and made available to the
US-RSE Association members. The tallies of any votes of the Steering Committee will also be made
available.

### Steering Committee Officers

Officers consist of a Chair, a Vice-Chair, Election Chair, and a Secretary, elected by the Steering
Committee. Officer elections will be held at the meeting following the election of new members to
the Steering Committee.

#### Chair

The Chair will set the agenda and preside over Steering Committee meetings or arrange for the Vice
Chair or another Steering Committee member to do so.

#### Vice-Chair

In the event of the absence of the Chair, the Vice-Chair will preside at meetings. The Vice-Chair

> will be responsible for ensuring that the bi-monthly newsletter will be sent out.  In addition, the

Reviewer 4: I'm not sure that we should specify bi-monthly, as at some point, we might want this to be monthly or even weekly.  Maybe we say "that a newsletters is sent out at least bi-monthly" or something even more general?

Reviewer 5: maybe that a newsletter will be sent out at least 3? times per year

Reviewer 3: or just "will be sent out regularly"?

Reviewer 4: would once every 2 years be ok?  That would be regular.

Reviewer 2: For a governance document, I'm not sure we even need to specify newsletter responsibility. Is it critical to include? It could very easily become someone else's responsibility, a community coordinator/manager or committee. All that said, if it says in, I'd be fine with some form of regular rather than bi-monthly.

Author: I agree with Ian, I don't think that needs to specified in a governance document. This iteration of the Steering Committee likes doing it bi-monthly, later iterations may choose monthly, or semi-annual, or even just to eliminate it in favor of more frequent blog/twitter/tiktok/whatever posts. I'm in favor of removing this entirely or if we must, include something like: "The Vice-Chair will be responsible for ensuring the community is kept informed of Association activities through regular newsletters, social media posts, etc."

Reviewer 3: Either of Lance's suggestions sounds fine to me.

Reviewer 4: the last suggestion is ok with me

Reviewer 2: I'm good with Lance's suggestion too.

Reviewer 6: I also like Lance's suggestion.

Author: OK, I've updated the document.

> Vice-Chair will prepare an annual report of Association activities.

Reviewer 4: ```suggestion
Vice-Chair will annually prepare a report of Association activities in the past year.
```

email.  Additionally, the Steering Committee Chair can resign by informing the full committee via
email. The resignation will take effect at the time specified in the notice, and, unless otherwise
specified, formal acceptance of the resignation is not necessary.

### Removal

Steering Committee members can be removed by a two-thirds (⅔) vote of the full Steering Committee.
In the event that that Committee is considering the removal of the Chair, the Vice-Chair will
preside.

### Vacancies

If a seat on the Steering Committee becomes vacant, the remaining members of the Steering Committee
may choose to appoint someone to fill the position for the remainder of the term. If the Steering
Committee chooses to fill the position, the replacement will be chosen by a majority vote of the
remaining members of the Steering Committee.  In the event that a majority vote cannot be achieved,
the seat shall remain open until the end of the seat's term.

## Committees, Working Groups, and Community Involvement

Organizational functions of the US-RSE Association will be managed by standing committees that may

> consist of one or more members of the Steering Committee and interested members and participants in

Reviewer 4: ```suggestion
consist of one or more members of the Steering Committee and interested members of and participants in
```

the Association. These organizational activities include communications (website, social media,
etc.), conference planning, and others to be determined by the Steering Committee.

Other activities, events, and efforts that have a fixed end date will be managed by working groups.
These working groups can and should be initiated by members at large through discussion with the
Steering Committee to ensure alignment with US-RSE priorities.  Working groups should have at least
three (3) US-RSE members as participants.

## Changes to Governance

> Minor revisions to this document can be made with the approval of at least two thirds of the current

Reviewer 4: ```suggestion
Minor revisions to this document can be made with the approval of at least two thirds (⅔) of the current
```

Reviewer 3: Will there be any technical issue with using a non-ASCII character here?

Author: I wouldn't think so these days.

> Minor revisions to this document can be made with the approval of at least two thirds of the current

Reviewer 5: ```suggestion
Minor revisions to this document can be made with the approval of at least two-thirds of the current
```

> steering committee members. Major changes to the governance structure or this document, will require

Reviewer 4: ```suggestion
steering committee members. Major changes to the governance structure or this document will require
```

a vote of the membership directly.

## Ratification

> This document was ratified by the following members of the First US-RSE Steering Committee on June
> XX, 2020:

Reviewer 2: It's a little bit harder here, but we still need an official date

```suggestion
This document was ratified by the following members of the First US-RSE Steering Committee on June
12, 2020:
```

Reviewer: I suggest we use the date this is merged into the repo. Whoever does the merge should just make one final commit to update this date.

* Jeffrey C. Carver
* Ian Cosden (chair)
* Charles Ferenbaugh
* Sandra Gesing (vice-chair)
* Chris Hill
* Daniel S. Katz

> * Christina Maimone

Reviewer 4: election chair?

> * Christina Maimone

Reviewer 5: ```suggestion
* Christina Maimone (election chair)
```
